iocage/lib/Jail.py

```diff
@@ -0,0 +1,409 @@
+import iocage.lib.JailConfig
+import iocage.lib.Network
+import iocage.lib.Storage
+import iocage.lib.Releases
+import iocage.lib.Release
+import iocage.lib.helpers
+
+import iocage.lib.ZFSBasejailStorage
+import iocage.lib.NullFSBasejailStorage
+import iocage.lib.StandaloneJailStorage
+
+import libzfs
+import subprocess
+import uuid
+
+class Jail:
+
+  def __init__(self, data = {}, zfs=None, host=None, logger=None):
```

Comment:
Good call. I haven't used a linter at all yet, but I will run a beautifier in one of the next commits. Thanks!